cmd/livepeer/livepeer.go

```diff
@@ -551,15 +551,16 @@ func setupTranscoder(n *core.LivepeerNode, lm *eth.LogMonitor) error {
 func txDataToVideoProfile(txData string) ([]lpmscore.VideoProfile, error) {
 	profiles := make([]lpmscore.VideoProfile, 0)
 
-	for i := 0; i < len(txData); i += lpcommon.VideoProfileIDSize {
+	for i := 0; i+lpcommon.VideoProfileIDSize < len(txData); i += lpcommon.VideoProfileIDSize {
 		txp := txData[i : i+lpcommon.VideoProfileIDSize]
 
 		p, ok := lpmscore.VideoProfileLookup[lpcommon.VideoProfileNameLookup[txp]]
 		if !ok {
-			glog.Errorf("Cannot find video profile for job: %v", txp)
-			return nil, core.ErrTranscode
+			// glog.Errorf("Cannot find video profile for job: %v", txp)
```

Comment:
Should we uncomment this so it bails out if there is no video profile for a particular video profile id?

Reply:
I was noticing some strange data getting passed through for whatever reason, so I thought we'd loosen the constraint to do the job if there is "any" valid video profile.  Any reason we won't want to do that?